packages/nvidia_nat_adk/src/nat/plugins/adk/tool_wrapper.py

```
@@ -0,0 +1,160 @@
+        Args:
+            func (Callable): The function to wrap.
+            nat_function (Function): The NAT Function to wrap.
+            name (str): The name of the function.
+            description (str): The description of the function.
+            input_schema (BaseModel): The Pydantic model defining the input schema.
+
+        Returns:
+            Callable[..., Any]: The wrapped function.
+        """
+        if func is None:
+            raise ValueError("Either 'func' or 'nat_function' must be provided.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)